editor/modes/visual-editor/block.js

```diff
@@ -51,13 +53,28 @@ import {
 	getMultiSelectedBlockUids,
 } from '../../selectors';
 
-const { BACKSPACE, ESCAPE, DELETE, ENTER } = keycodes;
+const { BACKSPACE, ESCAPE, DELETE, ENTER, TAB, F10, UP, LEFT, DOWN, RIGHT } = keycodes;
 
 function FirstChild( { children } ) {
 	const childrenArray = Children.toArray( children );
 	return childrenArray[ 0 ] || null;
 }
 
+function queryFirstTabbableChild( elem ) {
+	let i = 0;
```

Comment:
Unnecessary assignment since the `for` on the next line will set it to `0` anyways. Could also move entire declaration into the loop:

```js
for ( let i = 0; i < elem.childNodes.length; i++ ) {
```

Aside: `let` and `const` have different scoping implications than `var`, so conventions around assigning all variables top of function need not necessarily apply.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Scoping_rules_2